component/xload/comp/splitter.go

```diff
@@ -197,6 +197,13 @@ func (d *downloadSplitter) Process(item *common.WorkItem) (int, error) {
 
 	wg.Wait()
 
+	// update the last modified time
+	err = os.Chtimes(localPath, item.Atime, item.Mtime)
+	if err != nil {
+		log.Err("downloadSplitter::Process : Failed to change times of file %s [%s]", item.Path, err.Error())
+		operationSuccess = false
```

Comment:
Should we fail the preload if we fail to set the mod time, if user does not have permission it might fail so shall we just log and move on ?

Reply:
Done